mrp_subcontracting/models/mrp_production.py

```diff
@@ -67,6 +53,23 @@ def action_confirm(self):
                     self._create_external_procurement(wc_line))
         return res
 
+    @api.model
+    def _make_consume_line_from_data(self, production, product, uom_id, qty,
+                                     uos_id, uos_qty):
+        move_obj = self.env['stock.move']
+        move_id = super(MrpProduction, self)._make_consume_line_from_data(
+            production, product, uom_id, qty, uos_id, uos_qty)
+        if 'work_order' in self.env.context:
+            work_order = self.env.context.get('work_order')
+            if work_order.routing_wc_line.external:
+                picking_type = work_order.routing_wc_line.picking_type_id
+                vals = {'location_id': picking_type.default_location_src_id.id,
```

Comment:
Si el picking type es false, debería de coger las ubicaciones de forma standar, o sea, las que están en la cabecera de la OF. 
Por defecto lo hace así o sea que entiendo que sí que lo cogería aunque no he visto el código.